lib/supply/options.rb

```diff
@@ -33,19 +33,18 @@ def self.available_options
                                      optional: true,
                                      description: "Path to the directory containing the metadata files",
                                      default_value: (Dir["./fastlane/metadata/android"] + Dir["./metadata"]).first),
-        FastlaneCore::ConfigItem.new(key: :key,
```

Comment:
We could keep the KEY and ISSUER options, but just error out in the verify block, so it is clear to users who haven't changed their setup.